src/Parsers/ParserDropQuery.cpp

```diff
@@ -18,15 +18,17 @@ bool parseDropQuery(IParser::Pos & pos, ASTPtr & node, Expected & expected, cons
     ParserKeyword s_view("VIEW");
     ParserKeyword s_database("DATABASE");
     ParserToken s_dot(TokenType::Dot);
+    ParserToken s_comma(TokenType::Comma);
     ParserKeyword s_if_exists("IF EXISTS");
     ParserKeyword s_if_empty("IF EMPTY");
     ParserIdentifier name_p(true);
     ParserKeyword s_permanently("PERMANENTLY");
     ParserKeyword s_no_delay("NO DELAY");
     ParserKeyword s_sync("SYNC");
+    ParserNameList tables_p;
 
     ASTPtr database;
-    ASTPtr table;
+    ASTPtr database_and_tables;
```

Comment:
Are we now storing "database" twice (l. 30, l. 31)? We should try to avoid redundancy yere.

Reply:
It is not stored twice. When `DROP DATABASE...` is parsed, the database name will be stored in `database `. When `DROP TABLE...` is parsed, multiple table names will be stored in `database_and_tables `.